lib/screens/activity_log_screen.dart

```diff
@@ -46,11 +49,94 @@ class _ActivityLogScreenState extends State<ActivityLogScreen> {
   }
 
   Widget _buildActivityEntry(BuildContext context, ActivityEntry activity) {
-    return ListTile(
-      title: Text(activity.event),
-      subtitle: Text("On this device in Work."),
-      leading: Icon(Icons.file_download_outlined),
-      trailing: Text("2021-05-26"),
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 8.0),
+      child: Column(
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              _getActivityName(activity),
+              _getActivityDate(activity),
+            ],
+          ),
+          const SizedBox(height: 8.0),
+          Row(
+            children: [
+              _getActivityBasedIcon(activity),
+              const SizedBox(width: 8.0),
+              _getActivityDesc(activity),
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _getActivityName(ActivityEntry activity) {
+    return Text(
+      activity.event ?? '',
+      style: TextStyle(
+        fontWeight: FontWeight.bold,
+        color: Colors.black,
+        fontSize: 18,
+      ),
+    );
+  }
+
+  Widget _getActivityDate(ActivityEntry activity) {
+    return Text(
+      "${activity.timestamp?.day}/${activity.timestamp?.month}/${activity.timestamp?.year}",
```

Comment:
This formats the date disregarding of the apps localization. I don't mind this for the moment, but could you check if there is something like a localized print method for dates or an existing package which prints dates in various formats depending on the localization?

Reply:
i am not fully aware of the localization.

Reply:
I also don't know, how localization works in Flutter. Judged by the maturity of flutter I am confident some ready to use solutions exists, though. Could you please google for some best practices? Maybe this even is a standard feature of flutter? I don't think it should be too complicated to do this. Feel free to add any 3rd party packages, if this makes it easier.

Reply:
I try to find something.